### DISCUSSION TITLE
Can you change the "(Server Broadcast)" prefix?

### DISCUSSION THREAD

**Discussion by Mooshay105** [Pryaxis/TShock#2993] [Q&A] [answered]
Exactly the title.

**Comment by Arthri** [comment, marked as answer]
Short answer: no.

Long answer: modify the localization for the message with the ID `"(Server Broadcast) "`

https://github.com/Pryaxis/TShock/blob/5d585bb891ef6ed855ec3692c36712e64be3d867/TShockAPI/Commands.cs#L2020

**Comment by Mooshay105** [comment]
Ok, thanks for your help!

**Comment by sgkoishi** [comment]
Some plugins allow you to broadcast without the prefix. It's quite simple and you can try to make your own!
